ParamManagerCipher.pas

```diff
@@ -69,7 +69,7 @@ class function TDefaultCipher.DecodeString(const Value: AnsiString): AnsiString;
 
   GetMem(Tmp, Length(Value) + 1);
   try
-    AnsiStrings.StrPCopy(Tmp, Value);
+    StrPCopy(Tmp, Value);
```

Comment:
This doesn't work in XE7: `[dcc32 Error] ParamManagerCipher.pas(72): E2251 Ambiguous overloaded call to 'StrPCopy'`

If `StrPCopy` is not present in `AnsiStrings` on XE2 I guess the only solution is to use an IFDEF.